requirements.txt

```diff
@@ -1,2 +1,4 @@
 zc.buildout==2.13.2
-setuptools==41.0.1
\ No newline at end of file
+setuptools==41.0.1
```

Comment:
we could bump this to 42.0.2 later, last supported 2.7 version. iirc 43 started breaking things with buildout?